torch/onnx/utils.py

```diff
@@ -166,9 +161,8 @@ def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=Fa
         # until the point where it is unpacked by listUnpack node.
         # This pass does a preprocess, and prepares the nodes such that enough context can be received
         # by the symbolic function.
+        torch._C._jit_pass_onnx_remove_inplace_ops_for_onnx(graph, module)
```

Comment:
It seems that you postpone`_jit_pass_onnx_remove_inplace_ops_for_onnx` and remove the pass `_jit_pass_onnx_prepare_inplace_ops_for_onnx`, which is no longer needed?

Reply:
I have merged the two passes into one.